tests/dialects/builtin_test.py

```diff
@@ -0,0 +1,30 @@
+from xdsl.dialects.builtin import *
+
+
+def test_DenseIntOrFPElementsAttr():
```

Comment:
Maybe name this something like `test_DenseIntOrFPElementsAttr_fp_type_conversion` as you are testing a specific part of the DenseIntOrFPElementsAttr.